README.md

```diff
@@ -1,2 +1,32 @@
- 
+# revanced-patches
+
+A repository of patches for use with ReVanced tooling
+
+# List of patches
+|💊Patch|📜Description|🎯Target package|🏹Target package version|
+|:-----:|:-----------:|:--------------:|:----------------------:|
+|Background Play|Enable playing music in the background.|`com.google.android.apps.youtube.music`|5.03.50|
+|Exclusive Audio Playback|Add the option to play music without video.|`com.google.android.apps.youtube.music`|5.03.50|
+|Codecs Unlock|Enables more audio codecs. Usually results in better audio quality but may depend on song and device.|`com.google.android.apps.youtube.music`|5.03.50|
+|TasteBuilder Remover|Removes the "Tell us which artists you like" card from the Home screen. The same functionality can be triggered from the settings anyway.|`com.google.android.apps.youtube.music`|5.03.50|
+|Upgrade Button Remover|Remove the upgrade tab from the pivot bar in YouTube music.|`com.google.android.apps.youtube.music`|5.03.50|
+|Video Ads|Patch to remove ads in the YouTube video player.|`com.google.android.youtube`|17.25.34|
+|General Ads|Patch to remove general ads in bytecode.|`com.google.android.youtube`|17.25.34|
+|Hide Infocard Suggestions|Hides infocards in videos.|`com.google.android.youtube`|17.25.34|
+|Enable Debugging|Enable app debugging by patching the manifest file|`com.google.android.youtube`|all|
+|Custom Playback Speed|Allows to change the default playback speed options.|`com.google.android.youtube`|17.25.34|
+|Microg Support|Patch to allow YouTube ReVanced to run without root and under a different package name.|`com.google.android.youtube`|17.25.34|
+|Hdr Max Brightness|Set brightness to max for HDR videos in fullscreen mode.|`com.google.android.youtube`|17.25.34|
+|Seekbar Tapping|Enable tapping on the seekbar of the YouTube player.|`com.google.android.youtube`|17.25.34|
+|Fenster Swipe Controls|volume and brightness swipe controls|`com.google.android.youtube`|17.25.34|
+|Custom Branding|Change the branding of YouTube.|`com.google.android.youtube`|all|
+|Premium Heading|Show the premium branding on the the YouTube home screen.|`com.google.android.youtube`|all|
+|Old Quality Layout|Enable the original quality flyout menu.|`com.google.android.youtube`|17.25.34|
+|Hide Shorts Button|Hide the shorts button.|`com.google.android.youtube`|17.25.34|
+|Hide Cast Button|Patch to hide the cast button.|`com.google.android.youtube`|all|
+|Minimized Playback|Enable minimized and background playback.|`com.google.android.youtube`|17.25.34|
+|Disable Fullscreen Panels|Disable comments panel in fullscreen view.|`com.google.android.youtube`|17.25.34|
+|Disable Create Button|Disable the create button.|`com.google.android.youtube`|17.25.34|
+|Hide Watermark|Hide the creator's watermark on video's.|`com.google.android.youtube`|17.25.34|
+|Amoled|Enables pure black theme.|`com.google.android.youtube`|17.25.34|
+|Hide Autoplay Button|Disable the autoplay button.|`com.google.android.youtube`|17.25.34|
```

Comment:
This should not be pushed by anyone. The workflow would trigger the task which would trigger a commit which skips CI.

Reply:
Fixed.